src/status_im2/contexts/quo_preview/standart_auth/wrong_keycard.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im2.contexts.quo-preview.standart-auth.wrong-keycard
+  (:require [quo2.core :as quo]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [utils.re-frame :as rf]
+            [status-im2.common.standard-auth.keycard.view :as standard-auth]))
+
+(def descriptor
+  [{:key     :theme
```

Comment:
I'm not sure why we have this `theme` field in the descriptor, because the component `wrong-keycard-sheet` is not themeable according to your PR (no `theme` arg is taken).

Same for the other preview namespaces, none of them need the `theme` field in the descriptor.